package.json

```diff
@@ -34,6 +34,7 @@
 		"@babel/polyfill": "^7.4.4",
```

Comment:
Can we remove "@babel/polyfill" now, since we switch to core-js?

Reply:
Yes! :)

Reply:
Done